src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/BrokerProxy.cs

```diff
@@ -477,6 +477,9 @@ private Bundle GetBrokerOptions(AuthenticationRequest request)
             brokerOptions.PutString(BrokerConstants.AccountLoginHint, username);
             brokerOptions.PutString(BrokerConstants.AccountName, username);
 
+            brokerOptions.PutString(BrokerConstants.CallerInfoPackage, mContext.PackageName);
```

Comment:
What will happen with non-recent version of the broker (before the Android breaking change in the broker). Will these parameters be just ignored?

Reply:
good point. is it possible to test that out somehow?Does the broker have a version or similar where we can test for this? 

---
In reply to: [164066081](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/960#discussion_r164066081) [](ancestors = 164066081)

Reply:
yes, it will be ignored, As i understand  that change was done 6 or 7 broker releases ago, not sure that I can download it from Google play,